src/main/java/uk/gov/ons/census/casesvc/model/entity/Case.java

```diff
@@ -21,6 +21,8 @@
 
   @Column private UUID caseId;
 
+  @Column private String caseType;
```

Comment:
I guess it doesn't really matter, but any reason why this is top of the list? I would've put it after addressType, or maybe at the end

Reply:
Moved close to addressType.